.github/workflows/codesee-arch-diagram.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```

Comment:
Please remove this workflow. Any CI changes should be proposed in separate PR. But I'm not sure that we ready to add something like this now.